themes/bubble-wrap-icons.yaml

```diff
@@ -723,299 +723,300 @@ textures:
     mapzen_icon_library_shields_usa:
         url: images/bubble-wrap-shields-usa@2x.png
         filtering: mipmap
-        density: 5
+        density: 2
```

Comment:
The default icons density should also be 2? 

- https://github.com/tangrams/bubble-wrap/pull/270/files#diff-131e16e219854a71b9922916954b6624L45